addons/sale/wizard/sale_order_discount.py

```diff
@@ -121,7 +123,7 @@ def _create_discount_lines(self):
                         taxes=taxes,
                         description=_(
                             "Discount: %(percent)s%%",
-                            percent=self.discount_percentage*100
+                            percent=float_repr(self.discount_percentage * 100, discount_dp)
```

Comment:
Final nit:

```suggestion
                            percent=float_repr(self.discount_percentage * 100, discount_dp),
```
Not sure if ANV has already given you access to [RD-Sales Coding style conventions](https://www.odoo.com/odoo/knowledge/8790), but generally a comma should be added to the last argument, so we can add it here with the line getting modified regardless.